src/videoSyncManager.ts

```diff
@@ -67,12 +67,12 @@ export class VideoSyncManager {
     });
     this._eventManager.listen(this._mainPlayer, EventType.SEEKING, () => {
       this._logger.debug(`syncEvents :: seeking main player to to ${this._mainPlayer}`);
-      this._secondaryPlayer!.pause();
+      this._secondaryPlayer.pause();
     });
     this._eventManager.listen(this._mainPlayer, EventType.SEEKED, () => {
       this._logger.debug(`syncEvents :: seeked main player to ${this._mainPlayer.currentTime}`);
       if (this._mainPlayer.paused) {
-        this._secondaryPlayer.pause();
+        this._seekSecondaryPlayer(0);
       } else {
         this._secondaryPlayer.play();
       }
```

Comment:
_seekSecondaryPlayer already has this logic if checking if paused the pause else play.
So why not just call this._seekSecondaryPlayer(0)

Reply:
done